starter-code/src/clue.js

```diff
@@ -85,10 +85,122 @@ name: Patio
 */
 
 // Characters Collection
-const charactersArray = [];
+const charactersArray = [
+  {
+    nickName: "mrGreen",
+    first_name: "Jacob",
+    last_name: "Green",
+    color: "green",
+    description: "He has a lot of connections",
+    age: 45,
+    image:
+      "https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg",
+    occupation: "Entrepreneur"
+  },
+  {
+    nickName: "drOrchid",
+    first_name: "Doctor",
+    last_name: "Orchid",
+    color: "white",
+    description: "PhD in plant toxicology. Adopted daughter of Mr. Boddy",
+    age: 26,
+    image: "http://www.radiotimes.com/uploads/images/Original/111967.jpg",
+    occupation: "Scientist"
+  },
+  {
+    nickName: "profPlum",
+    first_name: "Victor",
+    last_name: "Plum",
+    color: "purple",
+    description: "Billionaire video game designer",
+    age: 22,
+    image:
+      "https://66.media.tumblr.com/ee7155882178f73b3781603f0908617c/tumblr_phhxc7EhPJ1w5fh03_540.jpg",
+    occupation: "Designer"
+  },
+  {
+    nickName: "missScarlet",
+    first_name: "Kasandra",
+    last_name: "Scarlet",
+    color: "red",
+    description: "She is an A-list movie star with a dark past",
+    age: 31,
+    image: "https://www.radiotimes.com/uploads/images/Original/111967.jpg",
+    occupation: "Actor"
+  },
+  {
+    nickName: "mrsPeacock",
+    first_name: "Eleanor",
+    last_name: "Peacock",
+    color: "blue",
+    description:
+      "She is from a wealthy family and uses her status and money to earn popularity",
+    age: 36,
+    image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+    occupation: "Socialité"
+  },
+  {
+    nickName: "mrMustard",
+    first_name: "Jack",
+    last_name: "Mustard",
+    color: "yellow",
+    description:
+      "He is a former football player who tries to get by on his former glory",
+    age: 62,
+    image:
+      "https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg",
+    occupation: "Retired Football player"
+  }
+];
 
 // Rooms Collection
-const roomsArray = [];
+const roomsArray = [
+  "Dining Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio"
+];
 
```

Comment:
So here you were supposed to create an array with OBJECTS inside. So each room object has a key "name". Otherwise you couldn't access mysteryEnvelope.room.name in your last iteration. Jasmin is not testing this last function with the arrays you created, instead "she" uses her own, so thats why you still pass the test;)